### PR TITLE
Apollo guide

### PR DESCRIPTION
**Pull request by tmeasday**
First pass, basically an outline.

### CODE REVIEW

**Comment by tmeasday** [comment]
Not sure why I didn't stick to the outline format, whatever

**Comment by tmeasday** [comment]
I'm going to work on making it happen in Todos now.

**Comment by lorensr** [comment]
Looks good 😊 I'm thinking it might be helpful to explain in equivalents format? https://github.com/meteor/guide/pull/443

Also, thoughts on adding advanced differences between it and pubsub/livedata? See Advanced section from ^

File: content/apollo.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## Apollo and MongoDB
+
+ - Another note about choices of DBs and RESTful APIs.
+ - Note that at this point Meteor's user system still uses Mongo behind the scenes, so if you want to use the `accounts-` packages, you will need a MongoDB server
+
+### Writing Mongo-based resolvers
+
+ - We can use Meteor's standard `Mongo.find/One()` calls
+ - We can also use `context.userId` for authentication, transparently
```
Comment (lorensr) [inline content/apollo.md]: I'm thinking "transparently" will have unclear meaning to many

Comment (tmeasday) [inline content/apollo.md]: Got it.

**Comment by lorensr** [comment]
Someone published these helpful articles we could reference... 😊

https://www.discovermeteor.com/blog/should-you-switch-to-apollo/
https://www.discovermeteor.com/blog/apollo-subscriptions-meteor-users/

**Comment by filipenevola** [comment]
We can't merge as the CLA is not signed.

**Pull request closed without merging**